Mathlib/Order/Birkhoff.lean

```diff
@@ -223,11 +238,15 @@ lemma birkhoffFinset_injective : Injective (birkhoffFinset α) :=
 
 end LatticeHom
 
+variable [DecidableEq α] [@DecidablePred α SupIrred]
```

Comment:
I doubt this serves any purpose at this point, since the lemma below shadows this α with its own. What happens when you remove this line?

Reply:
Good catch, thanks!